test/client/nodes_test.rb

```diff
@@ -41,7 +41,7 @@
   it "gets the hot threads for the node(s)" do
     str = $client.nodes.hot_threads :read_timeout => 2
     assert_instance_of String, str
-    assert_match %r/cpu usage by thread/, str
+    assert_match %r/:::/, str
```

Comment:
It appears that that commit targets ES 2.0, so we would need to bump from 1.4.0 to 2.0 for Travis, right? I'm all for making the test as accurate as possible. Silly non-JSON format...